src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Account/NoAgreement.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@model SFA.DAS.ProviderApprenticeshipsService.Web.Models.AccountHomeViewModel
+
+@{
+    ViewBag.Title = @Model.ProviderName;
+    ViewBag.PageId = "account-noagreement";
+}
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <h1 class="heading-xlarge">Provider agreement</h1>
+        <p>[Content placeholder]</p>
```

Comment:
i'm talking to sarah and also the fcs team about the flow and content. we'll need to see where the new pages sit to help those discussions so can be merged with placeholders for now